src/Mvc/omnisharp.json

```diff
@@ -0,0 +1,18 @@
+{
+  "RoslynExtensionsOptions": {
+    "enableAnalyzersSupport": true
+  },
+  "FormattingOptions": {
+    "enableEditorConfigSupport": true
+  },
+  "RenameOptions": {
+    "RenameInComments": true,
+    "RenameOverloads": true,
+    "RenameInStrings": true
+  },
+  "msbuild": {
+    "MSBuildSDKsPath": "..\\..\\.dotnet",
```

Comment:
We usually expect to use the `dotnet` in the path. Can we not specify this?

Reply:
This points to preview SDK that has been downloaded in the root `.dotnet` folder!

Reply:
Sure, but with VS you assume you've launched it with startvs or run activate so that the dotnet that's in the path is the right one. We should do the same here.

Reply:
yeah, just like we have instructed people to use `startvs.cmd` in the same way we will ask people to open repo in VSCode using `startvscode.cmd` which I have included in the PR as well!